src/main/conf/ds-cumulus-export-default-mapping.yml

```diff
@@ -96,10 +99,48 @@ digisam:
           replacement: "https://kb-images.kb.dk/DAMJP2/$1/"
           lineBreakIsMulti: false
 
-        - source: "Horizontal Pixels"
+        - source: "Horizontal Pixels"  # COP: part of Dimension
           dest: "width_pixels"
           destType: "long"
 
-        - source: "Vertical Pixels"
+        - source: "Vertical Pixels"  # COP: part of Dimension
           dest: "height_pixels"
           destType: "long"
+
+        - source: "Record Name"  # COP: Id
+          dest: "record_name"
```

Comment:
If this is the "real" ID for the record, it should be used instead of the `uuid` for the `id` field. If we want to have both, so that we can preserve the ID used for internal lookups in Cumulus, we could add a field such as `source_id`.

Reply:
keeping record_name